src/model_inspector/input_tester.py

```diff
@@ -1,28 +1,29 @@
 "A class for testing input shapes on a PyTorch model."
 from typing import List, Optional, Dict
-from src.model_inspector.utils import is_defined_shape, output_to_shape_dict
+from model_inspector.utils import is_defined_shape, output_to_shape_dict
```

Comment:
why is it no longer necessary to use the src root directory?

Reply:
I moved the test_local.py into src because the imports were acting up when I ran make file